src/basic/cgroup-util.h

```diff
@@ -78,6 +78,7 @@ bool cpu_accounting_is_cheap(void);
 #define CGROUP_WEIGHT_MAX UINT64_C(10000)
 #define CGROUP_WEIGHT_DEFAULT UINT64_C(100)
 
+#define CGROUP_LIMIT_INVALID ((uint64_t) -2)
```

Comment:
i am not a fan of this i must say, having a definition called INVALID between a definition MIN and one called MAX sounds weird, in particular as this is accessible from the client side...

I think the better approach is to just add a boolean to CGroupContext that declares whether the limit was configured already or not.